node/cli/src/service.rs

```diff
@@ -144,7 +144,7 @@ construct_service_factory! {
 			}},
 		LightImportQueue = AuraImportQueue<Self::Block>
 			{ |config: &FactoryFullConfiguration<Self>, client: Arc<LightClient<Self>>| {
-					import_queue(
+					import_queue::<_, _, _, Pair>(
```

Comment:
i would find this more explicit as `ed25519::Pair`. some with above.

Reply:
Makes sense, will fix.

Reply:
Fixed.